frontend/src/Settings/General/HostSettings.js

```diff
@@ -89,6 +90,21 @@ function HostSettings(props) {
         />
       </FormGroup>
 
+      <FormGroup
+        advancedSettings={advancedSettings}
+        isAdvanced={true}
+      >
+        <FormLabel>Application URL</FormLabel>
+
+        <FormInputGroup
+          type={inputTypes.TEXT}
+          name="applicationUrl"
+          helpText="This applications external URL including http(s):// and optional port and/or URL base"
```

Comment:
`This application's` or `Sonarr's`.

`including http(s)://, port and URL base`

Reply:
Done.